cmd/skunk/adapter/in/peer/peer.go

```diff
@@ -219,23 +215,33 @@ func (p *Peer) readMessage(conn *websocket.Conn, address string) (string, error)
 
 // ReadMessages starts readMessage for every conn in readConns in the readRate interval.
 func (p *Peer) ReadMessages(messageCh chan<- string, errorCh chan<- error) {
```

Comment:
Maybe add a few comments.

Reply:
done in: [130f30a](https://github.com/scherzma/Skunk/pull/33/commits/130f30aca6934913c3b65c19c3710094c3592fd3)